service/service.go

```diff
@@ -98,9 +98,9 @@ func New(config Config) (*Service, error) {
 			K8sClient:    k8sClient,
 			K8sExtClient: k8sExtClient,
 
+			IndexNamespace: config.Viper.GetString(config.Flag.Service.AppCatalog.IndexNamespace),
 			ProjectName:    config.ProjectName,
 			WatchNamespace: config.Viper.GetString(config.Flag.Service.Kubernetes.Watch.Namespace),
-			IndexNamespace: config.Viper.GetString(config.Flag.Service.AppCatalog.IndexNamespace),
```

Comment:
Alphabetical sorting is nice, and it was right before.

Reply:
I think the current status is correct since it follows the order of `Config`?

Reply:
Now the ordering looks OK.

Reply:
All good